### PR TITLE
0.5 json stable under load

### PR DESCRIPTION
**Pull request by eupakhomov**

### CODE REVIEW

**Comment by eupakhomov** [comment]
@sumlin I am trying to avoid to spend double effort making tests for not yet synchronized codebase. I tried to create pull requests in a way to add functionality incrementally to make it easier to merge. Once we are in sync I will migrate tests (which I already have done for standalone parts like utilities).

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java [hunk 1]
```diff
@@ -40,6 +46,11 @@
  */
 public class JSONClient implements IClientAPI {
 
+    private static final Logger logger = LoggerFactory.getLogger(JSONClient.class);
+
+    public final Draft draftOcppOnly =
+            new Draft_6455(Collections.emptyList(),
```
Comment (TVolden) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java]: Should be initiated in composite root (read the constructor).

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java [hunk 2]
```diff
@@ -30,16 +30,26 @@ of this software and associated documentation files (the "Software"), to deal
...
 
 public class JSONServer implements IServerAPI {
 
-    private WebSocketListener listener;
-    private Server server;
-    private FeatureRepository featureRepository;
+    private static final Logger logger = LoggerFactory.getLogger(JSONServer.class);
+
+    public final Draft draftOcppOnly =
+            new Draft_6455(Collections.emptyList(),
```
Comment (TVolden) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java]: Should be initiated in composite root (read the constructor).

**Comment by TVolden** [comment]
The IC shows a compile issue:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project v1_6: Compilation failure
[ERROR] /home/travis/build/ChargeTimeEU/Java-OCA-OCPP/ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java:[96,27] cannot find symbol
[ERROR]   symbol:   class SSLContext
[ERROR]   location: class eu.chargetime.ocpp.JSONClient
```

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java [hunk 3]
```diff
@@ -4,9 +4,16 @@
...
 import eu.chargetime.ocpp.model.Request;
+import eu.chargetime.ocpp.wss.BaseWssSocketBuilder;
+import eu.chargetime.ocpp.wss.WssSocketBuilder;
+import org.java_websocket.drafts.Draft;
+import org.java_websocket.drafts.Draft_6455;
+import org.java_websocket.protocols.Protocol;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import javax.net.ssl.SSLContext;
```
Comment (TVolden) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java]: Reverting the external API changes makes this line necessary again. A merge problem. :)

Comment (eupakhomov) [reply ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java]: That's true. Fixed.

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java (hunk 1 above)
Comment (eupakhomov) [reply ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java]: Initialization moved to constructor.

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java (hunk 2 above)
Comment (eupakhomov) [reply ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java]: Initialization moved to constructor.

**Pull request merged**